SimTKmath/Optimizers/src/c-cmaes/cmaes.c

```diff
@@ -99,9 +99,13 @@
   14/10/18: splitted cmaes_init() into cmaes_init_para() and cmaes_init_final(),
             such that parameters can be set also comparatively safely within the
             code, namely after calling cmaes_init_para(). The order of parameters
+<<<<<<< HEAD
             of readpara_init() has changed to be the same as for cmaes_init().
   14/11/25  fix warnings from Microsoft C compiler (sherm)
+=======
+            of readpara_init() has changed to be the same as for cmaes_init(). 
   14/11/26: renamed exported symbols so they begin with a cmaes_prefix.
+>>>>>>> bd5aeb2a21fbc437f1b322610022f7cb1ba6a9db
```

Comment:
Is this merge conflict in the c-cmaes codebase?

Reply:
Yeah: https://github.com/CMA-ES/c-cmaes/blob/master/src/cmaes.c#L108

Reply:
Seems to have been pulled in ~6 years ago:

```
ffdc1025 (Nikolaus Hansen    2014-12-10 19:49:29 +0100  102) <<<<<<< HEAD
97ada5f3 (Michael Sherman    2014-11-25 16:23:50 -0800  103)             of readpara_init() has changed to be the same as for cmaes_init().
97ada5f3 (Michael Sherman    2014-11-25 16:23:50 -0800  104)   14/11/25  fix warnings from Microsoft C compiler (sherm)
ffdc1025 (Nikolaus Hansen    2014-12-10 19:49:29 +0100  105) =======
a7adbb3e (Nikolaus Hansen    2014-10-19 15:16:39 +0200  106)             of readpara_init() has changed to be the same as for cmaes_init(). 
23d8ffc1 (Christopher Dembia 2014-11-26 14:38:32 -0800  107)   14/11/26: renamed exported symbols so they begin with a cmaes_prefix.
ffdc1025 (Nikolaus Hansen    2014-12-10 19:49:29 +0100  108) >>>>>>> bd5aeb2a21fbc437f1b322610022f7cb1ba6a9db
```